validation/src/main/scala/hmda/validation/ValidationStats.scala

```diff
@@ -22,7 +23,14 @@ object ValidationStats {
   case class ValidationStatsState(stats: Seq[SubmissionStats] = Nil) {
     def updated(event: Event): ValidationStatsState = event match {
       case SubmissionStatsAdded(s) =>
-        ValidationStatsState(stats :+ s)
+        val matchingSubs = stats.filter(stat => stat.id == s.id)
+        if (matchingSubs.isEmpty) {
+          ValidationStatsState(stats :+ s)
+        } else {
+          val oldSub = matchingSubs.head
+          val newSub = oldSub.copy(id = s.id, totalLars = oldSub.totalLars + s.totalLars, taxId = oldSub.taxId + s.taxId)
```

Comment:
This was my approach to modifying a currently existing `SubmissionStats` object.  When the Tax ID is found, [here](https://github.com/cfpb/hmda-platform/pull/939/files#diff-bf281a4df82187ea62138ba687b8007eR152), any unneeded fields are filled with an empty string or zero (in this case, a zero for the LAR count).  Later, once the total LAR count has been calculated [here](https://github.com/cfpb/hmda-platform/pull/939/files#diff-6ca062a943a4a1d03f3bddb0618aa3e9R51), an empty string is sent in place of the tax ID.  If this is confusing or unwanted, I can modify the approach.